### ISSUE TITLE
Add link to about page at the header

### ISSUE THREAD

**Issue by morchickit** [hasadna/Open-Knesset#519]
Currently, link to about page is on the bottom. Add link somewhere in the header - Either to the tool bar or on the top left corner

**Comment by alonisser** [comment]
in the right side there is already a logo and link to open knesset, maybe one of them should lead to the about page? since currently both lead to the same base oknesset url?

**Comment by morchickit** [comment]
Yes, I thought they are both the same link. Alright. I'll change it.
